classes/mix.bbclass

```diff
@@ -165,7 +184,7 @@ do_install() {
     ln -s ${REL_NAME} "${D}/${APP_PREFIX}/${APPNAME}/${APPVERSION}/bin/rc"
 
     install -m 0755 -d "${D}/${SYSCONFIG_PREFIX}"
-    install -m 0644 ${S}/config/${APPNAME}.conf ${D}/${SYSCONFIG_PREFIX}/${APPNAME}.conf
+    install -m 0644 ${S}/config/${APPNAME}.conf ${D}${CONFFILE}
     echo >> ${D}/${SYSCONFIG_PREFIX}/${APPNAME}.conf
```

Comment:
this path and the paths in the fallowing lines need to be replaced  with ${CONFFILE}